src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,311 @@
+
+    <blockquote class="twitter-tweet"><p lang="en" dir="ltr">Excited to see the buzz and vibrant community around <a href="https://twitter.com/libp2p?ref_src=twsrc%5Etfw">@libp2p</a> at <a href="https://twitter.com/hashtag/devcon4?src=hash&amp;ref_src=twsrc%5Etfw">#devcon4</a>. Once started by <a href="https://twitter.com/protocollabs?ref_src=twsrc%5Etfw">@protocollabs</a>, this modular, open P2P networking stack has become an asset and public good for decentralised infrastructure.<br><br>Ethereum, let’s <a href="https://twitter.com/hashtag/buidl?src=hash&amp;ref_src=twsrc%5Etfw">#buidl</a> the future of P2P networks together ❤️✌️ <a href="https://t.co/PCaMbJZK3s">pic.twitter.com/PCaMbJZK3s</a></p>&mdash; raulvk (@raulvk) <a href="https://twitter.com/raulvk/status/1058712044277510144?ref_src=twsrc%5Etfw">November 3, 2018</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+    - libp2p talks at the [Web3 Summit](https://www.youtube.com/watch?v=SWdMdN9eGqU), [EthCC 2](https://www.youtube.com/watch?v=yPOROKQRFPQ), and [CrossLink Taipei](https://www.youtube.com/watch?v=B7ecKMtNkXs)
+    - A new [libp2p devgrants program](https://github.com/libp2p/devgrants) kicked off at EthBerlinZwei with bounties for projects like [a Lua Wireshark dissector](https://github.com/ethberlinzwei/Bounties/issues/20) and [Noise handshakes in Go](https://github.com/libp2p/devgrants/blob/master/004-noise-handshake-implementations.md)
+3. **Analysis of the colloboration:**
+    The effort of integrating libp2p in Eth 2.0 also presented a valuable opportunity to reflect on what was going well in the collobration and areas that Protocol Labs could improve on.
+    In July 2019, a [study of libp2p and Eth2](https://discuss.libp2p.io/t/report-a-study-of-libp2p-and-eth2/229) was shared.
+    The report detailed struggles shared by Ethereum teams in the integration process and provided recommendations on how to overcome these issues.
+
+## Full Speed Ahead (2020-present) 🏎️
+
+The Eth 2.0 team started hosting dedicated "networking" meetings (started in Dec 2019) in addition to implementer calls. As before, libp2p core team members were key participants. These meetings were dedicated to [discussing progress and unblocking libp2p related endeavors](https://github.com/ethereum/eth2.0-pm/issues/124#issuecomment-579740557) (GossipSub paper & peer scoring, Noise, etc.)
+
+### GossipSub v1.1
+Updates to GossipSub and its formalization efforts also made significant headway.
+Protocol Labs:
+- Introduced [GossipSub v1.1](https://github.com/libp2p/specs/pull/254) which hardended the protocol to prevent a wide-range of attacks against it (see this [blog post](https://blog.ipfs.tech/2020-05-20-gossipsub-v1.1/) to learn more).
```

Comment:
```suggestion
- Introduced [GossipSub v1.1](https://github.com/libp2p/specs/pull/254), which hardened the protocol to prevent a wide range of attacks against it (see this [blog post](https://blog.ipfs.tech/2020-05-20-gossipsub-v1.1/) to learn more).
```